code/game/sound.dm

```diff
@@ -190,18 +190,29 @@ distance_multiplier - Can be used to multiply the distance at which the sound is
 			var/mob/M = m
 			M.playsound_local(M, null, volume, vary, frequency, null, channel, pressure_affected, S)
 
-/proc/play_soundtrack_music(var/datum/soundtrack_song/song, list/hearers = null, volume = 80, ignore_prefs = FALSE, play_to_lobby = FALSE, allow_deaf = TRUE, only_station = FALSE)
-	var/sound/S = sound(initial(song.file), volume=volume, wait=0, channel=CHANNEL_AMBIENT_MUSIC)
+// only_station:
+// 0: respect soundtrack option
+// 1: play to all
+// 2: play only on station
```

Comment:
define those

Reply:
uugh it's such a waste

Reply:
it's only usable for an admin proc call where defines arent usable anyway

Reply:
thank you ‼️